ios/FluentUI/Button/ButtonTokenSet.swift

```diff
@@ -225,36 +275,71 @@ public class ButtonTokenSet: ControlTokenSet<ButtonTokenSet.Tokens> {
 
 extension ButtonTokenSet {
     /// The value for the horizontal padding between the content of the button and the frame.
-    static func horizontalPadding(_ size: ButtonSizeCategory) -> CGFloat {
+    static func horizontalPadding(style: ButtonStyle, size: ButtonSizeCategory) -> CGFloat {
+        switch style.isFab {
```

Comment:
Why make this a switch statement? Why not just check `if style.isFab`? Ditto for the other static functions in this section.

This pattern is definitely useful for enums as a means of making sure you cover all values at compile time, but doing this for a boolean value seems like overkill.